### DISCUSSION TITLE
don't make search bar static make it sticky

### DISCUSSION THREAD

**Discussion by thenanosoft** [dailydotdev/daily#408] [Feature Requests]
![image](https://user-images.githubusercontent.com/67806445/147875114-e6fe8f02-a087-494e-9470-0fb8afbb7a31.png)

When we search and scroll through content, and go down, and then when we have to search for content again, we have to scroll up again and come up.

Make it sticky instead of static, so that scrolling up only once makes the search bar visible.

**Comment by idoshamun** [comment]
Thanks for this suggestion! We will consider it and I'll make sure to let you know :)
